Sample.java

```diff
@@ -1,58 +1,54 @@
 // Time Complexity :O(1)
-// Space Complexity :O(n)
+// Space Complexity :O(1 but 29/31 testcases passed)
```

Comment:
Kindly check your solution with the one done in the class. It should pass all the test cases.